Updating multi cell migration page (#6002)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/migration.md

pagination_prev: null
---

> dbt Labs is in the process of rolling out a new cell-based architecture for dbt Cloud. This architecture will be the foundation of dbt Cloud for years to come, and brings improved reliability, performance, and consistency to users of dbt Cloud.

Reviewer: ```suggestion
dbt Labs is in the process of rolling out a new cell-based architecture for dbt Cloud. This architecture provides the foundation of dbt Cloud for years to come, and brings improved reliability, performance, and consistency to users of dbt Cloud.
```

> Migrations are being scheduled on a per-account basis. If you have yet to receive any communication (either with a banner or by email) about a migration date, you don't need to take any action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.

Reviewer: Not sure if legal needs to look at this, but I suggest we more directly say that we're sending a communication.
```suggestion
We're scheduling migrations by account. When we're ready to migrate your account, you will receive a banner or email communication with your migration date. If you have not received this communication, then you don't need to take action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
```

Author: Not a legal doc, so they didn't need the review but the language looks good

> The migration process has been automated to the extent possible. However, you must take action to complete it. This document outlines the steps you must take to prevent service disruptions when your account is migrated to the cell-based architecture. 

Author: ```suggestion
Your account will be automatically migrated on its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
```

## Recommended actions

> It's highly recommended you take action on the following:

Author: ```suggestion
It's highly recommended you take action in the following areas:
```

> It's highly recommended you take action in the following areas:

Reviewer: ```suggestion
We highly recommended you take these actions:
```

> Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations will be voided during the migration and must be resent after it is complete.

Author: ```suggestion
- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations will be voided during the migration and must be resent after it is complete.
```

- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes will be lost during migration.
- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they will be lost during migration. If you lose critical logs older than 90 days during the migration, you will have to work with the dbt Labs Customer Support team to recover.

> ## Required action

Reviewer: ```suggestion
## Required actions
```

These actions are required to prevent users from losing access dbt Cloud:

- If you still need to, complete [Auth0 migration for SSO](/docs/cloud/manage-access/auth0-migration) before your scheduled migration date to avoid service disruptions. If you've completed the Auth0 migration, your account SSO configurations will be transferred automatically. 
- Update your IP allow lists. dbt Cloud will be using new IPs to access your warehouse post-migration. Allow inbound traffic from all of the following new IPs in your firewall and include them in any database grants:

    - `52.3.77.232` 
    - `3.214.191.130`
    - `34.233.79.135`

    Keep the old dbt Cloud IPs listed until the migration is complete.

## Post-migration​

> If all items on this checklist are completed, your dbt Cloud resources and jobs will continue working without interruption.

Reviewer: Making this more direct
```suggestion
Complete all of these items to ensure your dbt Cloud resources and jobs will continue working without interruption.
```

> You will have two URL login options:

Reviewer: Changing so it's less open-ended.
```suggestion
Use one of these two URL login options:
```
